### ISSUE TITLE
Invalid magic number while loading LoRAs

### ISSUE THREAD

**Issue by clarkmcc** [rustformers/llm#384]
I'm getting the following error when trying to load a LoRA.

```
InvalidMagic { path: "~/.chitchat/loras/alpaca-lora-7b.bin", magic: 4034b50 (PK) }
```

Is this because the LoRA is out of date, or is our ggml implementation out of date? I get the same error across 3-4 different LoRAs.

**Comment by LLukas22** [comment]
LoRA files should always start with the `ggla` magic. (See [here](https://github.com/ggerganov/llama.cpp/blob/master/convert-lora-to-ggml.py#L51)) Could you link to the LoRA file you were using? And could it be that you did use a pytorch file without converting it first?

**Comment by philpax** [comment]
I agree with Lukas - is it possible that you're using unconverted LoRAs? (Yes, it's rather confusing - we hope the situation will get better soon!)
